src/main/java/com/sarahhand/fractals/view/FractalsUI.java

```diff
@@ -98,25 +91,57 @@ public FractalsUI() {
 
 		colorSchemeComboBox = new JComboBox<>();
 		colorSchemeComboBox.setRenderer(new ColorSchemeListCellRenderer());
-		loadColorSchemes(fractalConfig.getColorScheme());
+		loadColorSchemes(MandelbrotConfig.DEAFAULT_CONFIG.getColorScheme());
```

Comment:
I don't think you should reference the MandelbrotConfig.DEFAULT_CONFIG, but instead events.getColorScheme(). That way you are guaranteed to get the color scheme used by the event handler / viewer.